packages/react-input/lib/react-input.js

```diff
@@ -27,7 +29,7 @@ const Input = ( {
 	}
 
 	return (
-		<div className="sui-form-field">
+		<div className={`sui-form-field ${errorStatus ? 'sui-form-field-error' : ''}`}>
```

Comment:
@Gowtham369 let's avoid unnecessary white space before/after a conditional class. Instead, add the class if you need it. As you can see in the following example:
```
${ className ? ' ' + className : '' }
```
Notice that before className I added an empty space between single quotes. This way the additional space that separates the class names is added once the condition meets.